test/smoketest/bootstrap_cvr.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+"""Bootstrap a Dominion CVR file
+Adapted from 
+ Bootstrap a very large data file | Tomochika Fujisawa's site
+ https://tmfujis.wordpress.com/2016/06/24/bootstrap-a-very-large-data-file/
+"""
+
+import sys
+import numpy
+
+for line in open(sys.argv[1], "r"):
+        cnt = numpy.random.poisson(lam=1)
+
+        if cnt == 0:
+                continue
+        else:
+                for i in range(cnt):
+                        sys.stdout.write(line)
```

Comment:
Is this file used elsewhere for something? It looks like the start of a CVR file generator.

Reply:
Yes - I used it (with some other head and tail commands) to make what we're testing with. Next on my plate is fixing the CVR ids as you requested.

Reply:
OK, cool.